tasks/quality_gates.py

```diff
@@ -196,6 +196,7 @@ def parse_and_trigger_gates(ctx, config_path=GATE_CONFIG_PATH):
             gate_states.append({"name": gate, "state": False, "error_type": "AssertionError", "message": str(e)})
         except InfraError as e:
             print(f"Gate {gate} flaked ! (InfraError)\n Restarting the job...")
+            traceback.print_exc()
```

Comment:
sad we're not using `logging`, otherwise we would have leveraged sth like:
```suggestion
            logging.exception(f"Gate {gate} flaked ! (InfraError)\n Restarting the job...")
```

Reply:
I was going to come up with the plan on how to improve SQG but would park this discussion at least until end of next week if it works for you